### PR TITLE
Added documentation on changes for Web API action filter HttpResponse handling

### PR DESCRIPTION
**Pull request by alistairjevans**

### CODE REVIEW

**Comment by alistairjevans** [comment]
Related to pull request 42 on Autofac.WebAPI.

https://github.com/autofac/Autofac.WebApi/pull/42

File: docs/integration/webapi.rst [hunk 1]
```diff
@@ -228,6 +228,31 @@ Autofac introduces the new interfaces to allow you to concentrate on implementin
 
 Another reason for creating the internal attribute wrappers is to support the ``InstancePerRequest`` lifetime scope for filters. See below for more on that.
 
+Setting the Response in an Autofac Action Filter
+------------------------------------------------
+
+In the same way as with standard Web API filters,  you are able to set the HttpResponse object in the
```
Comment (tillig) [inline docs/integration/webapi.rst]: It looks like it's an `HttpResponseMessage` not an `HttpResponse` [from the code](https://github.com/aspnet/AspNetWebStack/blob/ebef5b7d821b64ed5c48765d0caf6ce8a9bcfaf5/src/System.Web.Http/Controllers/HttpActionContext.cs#L82).

Comment (tillig) [reply docs/integration/webapi.rst]: There's another instance of that below.

File: docs/integration/webapi.rst [hunk 2]
```diff
@@ -228,6 +228,31 @@ Autofac introduces the new interfaces to allow you to concentrate on implementin
...
+      return Task.FromResult(0);
+    }
+
+    public void Task OnActionExecutedAsync(HttpActionExecutedContext actionExecutedContext, CancellationToken cancellationToken)
+    {
+    }
+  }
+
+To match the standard Web API behaviour, if you set the Response object, then no subsequent action filters will be 
+invoked. However, any action filters already invoked will have OnActionExecutedAsync called with the appropriate response populated.
```
Comment (tillig) [inline docs/integration/webapi.rst]: Don't forget the double-backticks to make `OnActionExecutedAsync` and `HttpResponseMessage` show up as keywords.

**Review by tillig — CHANGES_REQUESTED** [review]
A couple minor updates needed, otherwise looks good.

File: docs/integration/webapi.rst (hunk 1 above)
Comment (alistairjevans) [reply docs/integration/webapi.rst]: Corrected the name, added backticks.

File: docs/integration/webapi.rst (hunk 2 above)
Comment (alistairjevans) [reply docs/integration/webapi.rst]: Changed 'object' to 'property' to make it clear I mean the property on the context. Added backticks.

**Review by tillig — APPROVED** [review]
Looks great!

**Pull request merged**